aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -7,22 +7,22 @@
 namespace at {
 namespace native {
 
-QTensor quantize_linear(const RealTensor& self, double scale, int64_t zero_point) {
+QTensor quantize_linear_cpu(const RealTensor& self, double scale, int64_t zero_point) {
   auto quantizer = make_per_tensor_affine_quantizer(scale, zero_point);
   return quantizer->quantize(self);
 }
 
-RealTensor dequantize(const QTensor& self) {
+RealTensor dequantize_(const QTensor& self) {
```

Comment:
why does this have an underscore?  That typically means its inplace, and I don't think we want inplace dequantize.

Reply:
oh that's because of dispatch, see native_functions.yaml.
I'll change the name